src/js/other-websites/add_input_icon.js

```diff
@@ -162,6 +162,9 @@ async function addRelayIconToInput(emailInput) {
   relayIconBtn.id = "fx-relay-button";
   relayIconBtn.type = "button";
   relayIconBtn.title = browser.i18n.getMessage("pageInputIconGenerateNewAlias");
+  const makeNewAliasImagePath = browser.runtime.getURL('icons/make-new-alias.png');
```

Comment:
Q: Should this have a leading `/`, like L383 below:
```js
browser.runtime.getURL('/images/loader.svg');
```

Reply:
Fixed!